File: Taller1ED_2019-2/Cliente.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   Cliente.cpp
 * Author: Juan Pablo Martinez
 * 
 * Created on 18 de septiembre de 2019, 16:47
 */

#include "Cliente.h"

Cliente::Cliente(string nombre, int fondo) {
    this->nombre=nombre;
    this->fondos=fondo;
}

int Cliente::getFondos() {
    return this->fondos;
}

string Cliente::getNombre() {
    return this->nombre;
}

void Cliente::setFondo(int fondos) {
    this->fondos=fondos;
}

void Cliente::setNombre(string nombre) {
    this->nombre=nombre;
}

Cliente::~Cliente() {
}

File: patron_DAO/src/patron_dao/Cliente.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package patron_dao;

/**
 *
 * @author Juan Pablo Martinez
 */
public class Cliente {

    private int id;
    private String nombre;
    private String apellido;

    public Cliente() {
    }

    public Cliente(int id, String nombre, String apellido) {
        this.id = id;
        this.nombre = nombre;
        this.apellido = apellido;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    @Override
    public String toString() {
        return "Cliente{" + "id=" + id + ", nombre=" + nombre + ", apellido=" + apellido + '}';
    }
    

}
